plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimatorsFactory.kt

```diff
@@ -280,10 +280,19 @@ class CameraAnimatorsFactory internal constructor(mapDelegateProvider: MapDelega
 
     val size = mapTransformDelegate.getSize()
     val pixelRatio = mapTransformDelegate.getMapOptions().pixelRatio
-    val w0 = max(
-      (size.width - endPadding.left - endPadding.right) / pixelRatio,
-      (size.height - endPadding.top - endPadding.bottom) / pixelRatio
-    )
+    val w0 = if (size.width.toDouble() != endPadding.left + endPadding.right &&
+      size.height.toDouble() != endPadding.bottom + endPadding.top
```

Comment:
Is it better to use`>` to replace `!=`?

Reply:
I'd say no. We must exclude equality in order to avoid division by zero. However if paddings are set in a way that they in total are greater than size - it seems to be more of an issue in this logic but flyTo algo will work OK in that case.